plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -796,9 +796,9 @@ global proc int mayaUsdTranslatorExport (string $parent,
         mayaUsdTranslatorExport_EnableAllControls();
         mayaUsdTranslatorExport_SetFromOptions($currentOptions, 1, 1);
 
-        // Set visibility for anim widgets
+        // Set visibility for anim widgets (but do not update the start/end time to playback range
+        // as they will be set by the caller).
         mayaUsdTranslatorExport_AnimationCB();
-        mayaUsdTranslatorExport_AnimationRangeCB();

```

Comment:
As per discussion with design, they chose to remove this auto-update of the animation range (to the playback range) as it does not reflect the true values used by the operations other than export (such as Duplicate as Usd).